client/src/components/organisms/AdminRequestGroupBrowser.tsx

```diff
@@ -70,6 +73,21 @@ const AdminRequestGroupBrowser: FunctionComponent = () => {
         }
     }, [requestGroup]);
 
+    const deleteQuery = gql`
```

Comment:
Might be clearer to rename to deleteRequestGroupQuery?

Reply:
Renamed